dotnet-client-libraries/TestDataGenerationWithAutoFixture/TestDataGenerationWithAutoFixture/TestDataGenerationWithAutoFixture.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
Since the console app is doing nothing but printing "Hello World", would it be better to make this just a simple library by changing this line?

Reply:
Done, good point.